grafana/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # https://hub.docker.com/r/grafana/grafana/tags
-FROM grafana/grafana:8.5.16
+FROM grafana/grafana:8.5.21
```

Comment:
8.5.21 - seems not publically showing in grafana releases

Reply:
https://hub.docker.com/layers/grafana/grafana/8.5.21/images/sha256-5419f6affd2c34fce914bd1787985c0a27435d13c39dd535ba2121702e0609cb?context=explore